Makefile

```diff
@@ -3,62 +3,61 @@
 #
 all: binary
 
-# remove build artifacts
+
 .PHONY: clean
-clean:
+clean: ## remove build artifacts
 	rm -rf ./build/* cli/winresources/rsrc_* ./man/man[1-9] docs/yaml/gen
 
-# run go test
-# the "-tags daemon" part is temporary
 .PHONY: test
-test:
+test: ## run go test (the "-tags daemon" part is temporary)
```

Comment:
The note about `-tags daemon` is more for the reading of the file, not someone running tasks, can we remove that bit?